### DISCUSSION TITLE
Installed using docker-compose on Ubuntu 20.04 - can't login as admin

### DISCUSSION THREAD

**Discussion by dsaffie** [DependencyTrack/dependency-track#1448] [Q&A] [answered]
I just did a fresh install via docker-compose on Ubuntu Server 20.04. After installing I setup an nginx server to access it at a custom domain. If I hit the URL at 8080, I arrive at the Dependency Track login screen, but I cannot login. I tried the admin/admin initial login, but nothing happens; no error or any indication it worked or did not work. Am I doing something wrong?

**Comment by stevespringett** [comment]
The frontend UI needs to be able to access the API server. Open up the console in the developer tools in your browser and see if there's any warnings, etc.

**Comment by iamrahul127** [comment, marked as answer]
I think you have probably not updated APP_BASE_URL.  Get into _frontend_ container and update static/config.json with URL pointing to your api server.
